client/rest-high-level/src/main/java/org/elasticsearch/client/graph/GraphExploreResponse.java

```diff
@@ -94,62 +90,36 @@ public boolean isTimedOut() {
         return connections.values();
     }
 
+    public Collection<ConnectionId> getConnectionIds() {
+        return connections.keySet();
+    }
+
+    public Connection getConnection(ConnectionId connectionId) {
+        return connections.get(connectionId);
+    }
+
     public Collection<Vertex> getVertices() {
         return vertices.values();
     }
+
+    public Collection<VertexId> getVertexIds() {
```

Comment:
Could you explain why you added these? I'm not against it, but in general I think we should avoid adding them unless we're super sure we want them and I think we should javadoc them.

Reply:
this is a drawback of cross hlrc-xpack tests: xpack and hlrc responses are in different packages and have to add such accessors to be able to convert one instance into another

Reply:
Ah! I see that this data wasn't available because we just returned the `values`. That makes sense now. :+1: